Sources/Testing/Test.swift

```diff
@@ -210,17 +208,46 @@ extension Test {
     /// - ``Test/parameters``
     public var parameters: [Parameter]?
 
+    /// We can't store arbitrary traits on snapshots because they're not
```

Comment:
I'd like to see if we can come up with some general-purpose solution for arbitrary traits, but this seems like the best we can do right now.

The documentation comments for these properties should actually document them (i.e. duplicate the comments on the non-snapshot properties.) This note about what we can and can't encode can be a simple comment above the properties.